resources/views/packs/index.blade.php

```diff
@@ -27,7 +27,7 @@
                         <div class="beatmap-pack__name">{{ $pack->name }}</div>
                         <div class="beatmap-pack__details">
                             <span class="beatmap-pack__date">{{ json_date($pack->date) }}</span>
-                            <span>{{ osu_trans('beatmappacks.show.created_by', ['author' => "<strong>{$pack->author}</strong>"]) }}</span>
+                            <span>{!! osu_trans('beatmappacks.show.created_by', ['author' => "<strong>{$pack->author}</strong>"]) !!}</span>
```

Comment:
`author` should be escaped and wrapped in `HtmlString`, use `make_blade_safe`; same for the other file